Update Profile.tsx

Eeyore6 opened the pull request:
Add skeleton Profile UI, add a profile image, edit button, and information blocks(blocks have titles and can be editable, the data cannot be saved)

Eeyore6 committed:
Update Profile.tsx

Add skeleton Profile UI, add a profile image, edit button, and information blocks(blocks have titles and can be editable, the data cannot be saved)

mursal-ashraf commented:
> Add skeleton Profile UI, add a profile image, edit button, and information blocks(blocks have titles and can be editable, the data cannot be saved) 

Small request, can you put a screenshot up on this PR?

mursal-ashraf commented on frontend/src/components/Profile/Profile.tsx:
```diff
@@ -1,7 +1,55 @@
+                        </Button>
+                    </div>
+                    <div className="ml-6 flex flex-col justify-between w-3/4 space-y-4">
+                        <input name="name" value={formData.name} onChange={handleInputChange} className="h-20 bg-gray-300 rounded w-3/4 text-xl text-black pl-4" placeholder="Name" />
```
If you're interested, Material UI does have a TextField component. It could mean less code for us to write. Thoughts @KevinLu9?
https://mui.com/material-ui/react-text-field/

Eeyore6 replied:
I think this should be better to use the text component from MUI, I did not realise I can use that.
Here is the screenshot of current profile page.


![profile_1](https://github.com/mursal-ashraf/bankan/assets/100513480/d5fe0d7b-7b53-4214-82f6-67478f5cce6e)

mursal-ashraf replied:
Thanks Dongyi! If MUI components can be added, then PR is good to go! Page looks 👌

Eeyore6 replied:
Sure! I am going to add the MUI part and try to make the UI looks better. Gonna finish them before the Sunday

KevinLu9 replied:
Yea, i think we should try to use MUI whenever possible.

mursal-ashraf commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Create Basic Structure of profile page](https://trello.com/c/Po1lMLTL/35-create-basic-structure-of-profile-page)

Eeyore6 committed:
Profile -> Update with MUI material

Add new background color as the Figma is, add new MUI TextFields and replace the old ones.

Eeyore6 committed:
Profile Update Edit Button

Add a new pop-up window to the edit button which users could use to upload/replace pics

Eeyore6 committed:
Profile Adjustments

Resize the components

mursal-ashraf commented on frontend/src/components/Profile/Profile.tsx:
```diff
@@ -1,7 +1,132 @@
+    };
+
+    const handleUploadClick = () => {
+        console.log("Upload button clicked!");
```
Can you remove console.log's as well?

Eeyore6 replied:
Sure, and for further question about the footer, I may use the common file you created. This might need to be done after your work merged

mursal-ashraf replied:
Yeah we can do that in a future card/PR. For now we'll merge this in.

Eeyore6 replied:
All good, thanks for reviewing!

mursal-ashraf reviewed (approved):
Just removing console.log and then good to merge. Nice work 👍

Eeyore6 committed:
Profile.tsx Update

Remove the console.log in Function handleUploadClick().
